### DISCUSSION TITLE
Set chart location to a dedicated worksheet

### DISCUSSION THREAD

**Discussion by geoced** [dfinke/ImportExcel#1490] [Q&A] [answered]
Hello,

I've been trying to set the location of a chart such that it is not embedded as an object in an existing worksheet but sits on a dedicated worksheet (see dialog box in the screenshot below). It should look like this (fullscreen graph with no cells):

![image](https://github.com/dfinke/ImportExcel/assets/40609578/c77744a6-1e6b-4cc2-82bb-9a4f346f3b33)

But no matter what I try, it always ends up as an embedded object:

![image](https://github.com/dfinke/ImportExcel/assets/40609578/b5b027a3-462d-420f-99b2-755ef5dc607f)

Can this be achieved ?

**Comment by geoced** [comment, marked as answer]
Found it !

You have to create an ExcelChartSheet with this code:

```
$chartSheet = $xl.Workbook.Worksheets.AddChart('SheetTitle', [OfficeOpenXml.Drawing.Chart.eChartType]::Line)
$chart = $chartSheet.Drawings[0]
$chart.Title.Text = 'Chart Title'
$series = $chart.Series.Add($yRange, $xRange)
```

Would be nice if this option was offered by ImportExcel, e.g. as a [switch] parameter of Add-ExcelChart !

**Reply by dfinke** [reply]
Only so many switches can be offered. Sorry
